include/mesh/exodusII_io.h

```diff
@@ -60,7 +60,8 @@ class ExodusII_IO : public MeshInput<MeshBase>,
    */
   explicit
   ExodusII_IO (MeshBase & mesh,
-               bool single_precision=false);
+               bool single_precision=false,
+               const std::vector<std::string> & extra_integer_vars = {});
```

Comment:
I think I'd prefer not to change the constructor if possible since nothing else is really handled this way? Can we handle it similar to `ExodusII_IO::set_output_variables()`, where the user can pass in a reference to a vector of strings that tells which variables are extra integer variables?

Reply:
Will do.